integration-tests/src/query_params.test.js

```diff
@@ -91,6 +91,66 @@ describe("for a GET endpoint with no parameters", () => {
       last: "Last"
     });
   });
+
+  test("server will parse empty inputs on non optional parameters as empty strings", async () => {
+    const server = TestUtils.makeServer({
+      endpoint: testEndpoint,
+      handler: testHandler
+    });
+    const resp = await fetch(
+      `http://localhost:${server.address().port}/foo?first=&last=Last`
+    );
+    await resp.json();
+    expect(testHandler).toHaveBeenLastCalledWith({
+      first: "",
+      last: "Last"
+    });
+  });
+});
+
+const testNumEndpoint: Recouple.Endpoint<
+  {
+    x: number
+  },
+  number
+> = Recouple.endpoint()
+  .fragment("foo")
+  .queryParams({
+    x: T.number
+  });
+
+const testNumHandler = jest.fn(async () => {
```

Comment:
does this need to be different than `testHandler`?

Reply:
This isn't actually being used by the test (since we spy on the input to the function), but the types need to match.